### ISSUE TITLE
panic: runtime error: slice bounds out of range

### ISSUE THREAD

**Issue by wonderivan** [cjbassi/gotop#81]
panic: runtime error: slice bounds out of range

goroutine 22 [running]:
github.com/cjbassi/gotop/src/widgets.Processes(0xc00002ff70, 0x400dec8, 0x10)
	/ext-go/1/src/github.com/cjbassi/gotop/src/widgets/proc_unix.go:40 +0x875
github.com/cjbassi/gotop/src/widgets.(*Proc).update(0xc000160000)
	/ext-go/1/src/github.com/cjbassi/gotop/src/widgets/proc_unix.go:13 +0x26
github.com/cjbassi/gotop/src/widgets.NewProc(0x0)
	/ext-go/1/src/github.com/cjbassi/gotop/src/widgets/proc.go:63 +0x167
main.initWidgets.func3(0xc000093100)
	/ext-go/1/src/github.com/cjbassi/gotop/main.go:218 +0x22
created by main.initWidgets
	/ext-go/1/src/github.com/cjbassi/gotop/main.go:217 +0xb6

**Comment by vothanhkiet** [comment]
I also get the same error

  System Version:	macOS 10.14.1 (18B75)
  Kernel Version:	Darwin 18.2.0

**Comment by cjbassi** [comment]
I don't have a mac to test on, so can you run
```
ps -axo pid:10,comm:50,pcpu:5,pmem:5,args | head
```
and copy the output here?

**Comment by vothanhkiet** [comment]
Ok

```
ps -axo pid:10,comm:50,pcpu:5,pmem:5,args | head                                                                                                                   103ms  Thu Dec 13 21:17:38 2018
ps: pid:10: keyword not found
ps: comm:50: keyword not found
ps: pcpu:5: keyword not found
ps: pmem:5: keyword not found
ARGS
/sbin/launchd
/usr/sbin/syslogd
/usr/libexec/UserEventAgent (System)
/System/Library/PrivateFrameworks/Uninstall.framework/Resources/uninstalld
/usr/libexec/kextd
/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/FSEvents.framework/Versions/A/Support/fseventsd
/System/Library/PrivateFrameworks/MediaRemote.framework/Support/mediaremoted
/usr/local/opt/dnsmasq/sbin/dnsmasq --keep-in-foreground -C /usr/local/etc/dnsmasq.conf
/System/Library/CoreServices/appleeventsd --server
```

**Comment by cjbassi** [comment]
This panic is caused by a discrepancy between `ps` on Linux and osx, so I reverted some changes I had just made that weren't supported on osx. Can you try the master branch and let me know if it works you now?

**Comment by vothanhkiet** [comment]
I've just pulled the master branch, it works now

<img width="1440" alt="screen shot 2018-12-14 at 13 02 27" src="https://user-images.githubusercontent.com/4302582/49985992-ae5d4b00-ffa0-11e8-9067-24739b85ba5f.png">

**Comment by cjbassi** [comment]
@vothanhkiet Glad its working. Also just released v1.7.1 with the fix too.
